CONTRIBUTING.md

```diff
@@ -182,7 +182,7 @@ will be ignored.
 ### Publish release notes
 
 The following template should be used for creating GitHub release notes via
-[this form](https://github.com/mongodb/mongo-php-library/releases/new).
+[this form](https://github.com/mongodb/mongo-php-library/releases/).
```

Comment:
https://github.com/mongodb/mongo-php-library/releases/ is the publicly accessible page that lists releases for the project. https://github.com/mongodb/mongo-php-library/releases/new is actually the correct link to the form for _creating a new release_. That page is likely restricted to GitHub users with permissions to do so on the repository:

![image](https://user-images.githubusercontent.com/244663/145080909-2b5bd557-fb53-4848-b387-6019ec45b4f6.png)

Reply:
Oh Okay..Thank you for your quick review.